rules-reviewed/jboss-eap/websphere/websphere.windup.xml

```diff
@@ -0,0 +1,701 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="websphere_rules_general">
+    <rules>
+        <rule id="websphere_rules_general-7Apvvd">
+            <when>
+                <javaclass references="com.ibm.websphere.sib.api.jms.JmsTopicConnectionFactory" as="default"/>
```

Comment:
It's not a big deal, but I would like to see the 'as="default"' stuff removed. I think restating the default is a little confusing.